app/controllers/comments_controller.rb

```diff
@@ -1,4 +1,42 @@
 class CommentsController < AuthenticatedController
+  include ActionView::RecordIdentifier
   include ProjectScoped
-end
 
+  before_action :find_or_initialize_comment
```

Comment:
this isn't the standard `before_action` name we're now using. It also isn't the standard implementation for `set_comment` we're using elsewhere.